saga/inMemorySagaLog.go

```diff
@@ -69,3 +69,19 @@ func (log *inMemorySagaLog) GetMessages(sagaId string) ([]sagaMessage, error) {
 		return nil, nil
 	}
 }
+
+/*
+ * Returns all Sagas Started since this InMemory Saga was created
+ */
+func (log *inMemorySagaLog) GetActiveSagas() ([]string, error) {
+	log.mutex.RLock()
```

Comment:
(BTW, just googled an found this, https://github.com/golang/go/issues/6980 , which shows they know about it, and the cost is ~70ns)